### ISSUE TITLE
The color contrast ratio of focus indicator over the "Azure-Refresh", "Azure-Sign-in" and "Remove account" buttons is less than 3:1. : A11y_SQL Azure Data Studio_Azure account_AI4D

### ISSUE THREAD

**Issue by Mana-swi** [microsoft/azuredatastudio#26418]
### Tags:
#A11yTCS;#A11yMAS;#WCAG1.4.11;#Non-text Contrast;#A11ySev2;#Win32;#DesktopApp;#BM_SQL Azure Data Studio_Win32_Sep25;#SQL Azure Data Studio;
**"[[Check out Accessibility Insights!]](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0)) - Identify accessibility bugs before check-in and make bug fixing faster and easier."**
### Environment Details:
Application Name: SQL Azure Data Studio
Version: 1.52.0 (system setup)
VS Code: 1.82.0
Electron: 30.5.1
Chromium: 124.0.6367.243
Node.js: 20.16.0
V8: 12.4.254.20-electron.0
OS: Windows_NT x64 10.0.22621
### Repro Steps:
1.1. Launch Azure Data Studio.
2. Tab to Connections icon (if not already selected) from left bar.
3. tab to  to the 'Azure' header and enter the '+' icon.
4. tab to Azure header in the blade that opens up on the right side and enter on the "+" icon.
5. Observe that The color contrast ratio of focus indicator over the "Azure-Refresh", "Azure-Sign-in" and "Remove account" buttons is less than 3:1
### Actual:
The color contrast ratio of focus indicator over the "Azure-Refresh", "Azure-Sign-in" and "Remove account" buttons is less than 3:1

### Expected:
The color contrast ratio of focus indicator over the "Azure-Refresh", "Azure-Sign-in" and "Remove account" buttons should be greater than or equal to 3:1 w.r.t. it's background.

### User Impact:
Low vision users will not be able to identify and locate the focus on the controls as it's failing the minimum required color contrast w.r.t. it's background.
### Recommendations:
 [MAS Problems and Suggested Fixes](https://nam06.safelinks.protection.outlook.com/?url=https:%2f%2fmicrosoft.sharepoint.com%2fsites%2faccessibility%2fLists%2fMASProblemsAndSuggestedFixes%2fReviewed%2520Items.aspx&data=04%7c01%7cv-prajadhav%40microsoft.com%7cf642cb2fc53d4c4be38408d9d97cf216%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637779952946112692%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c3000&sdata=avswAOFlXEiWy%2fRT%2f%2fPGnurQzMGOUdSW7Z5Xp8ZioiQ%3D&reserved=0)
### MAS Reference:

- [MAS 1.4.11 – Non-text Contrast](https://aka.ms/MAS1.4.11)
- [MAS 1.4.11: Non-Text Contrast Guide](https://microsoft.sharepoint.com/sites/accessibility/SitePages/MAS-1.4.11-Non-Text-Contrast-Guide.aspx)

<img width="1139" height="604" alt="Image" src="https://github.com/user-attachments/assets/359dc48c-33a8-436d-a079-b1187ab0a47c" />

<img width="1149" height="604" alt="Image" src="https://github.com/user-attachments/assets/66d7290d-8324-449e-90c5-9fde4f73996e" />
